static/src/javascripts/projects/common/modules/commercial/creatives/commercial-component.js

```diff
@@ -103,13 +105,32 @@ define([
         }
     }
 
+    function initMostPopularCard(el) {
+        var height;
+        var $adSlot = $(el);
+        var $mostPopTabs;
+
+        if ($adSlot.hasClass('ad-slot--mostpop')) {
+            fastdom.read(function () {
+                height = $adSlot.dim().height;
+                $mostPopTabs = $('.js-most-popular-footer .tabs__pane');
```

Comment:
You can move this out of the Fastdom batch, querying the DOM does not trigger layout.